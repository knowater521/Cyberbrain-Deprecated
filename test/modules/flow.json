[
  {
    "code": "x = foo.func_in_foo()",
    "next": "cyberbrain.register(x)",
    "prev": "",
    "step_into": "a = bar.func_in_bar()",
    "returned_from": "return a"
  },
  {
    "code": "a = bar.func_in_bar()",
    "next": "return a",
    "prev": "x = foo.func_in_foo()",
    "step_into": "return 1",
    "returned_from": "return 1"
  },
  {
    "code": "return 1",
    "next": "",
    "prev": "a = bar.func_in_bar()",
    "step_into": "",
    "returned_from": ""
  },
  {
    "code": "return a",
    "next": "",
    "prev": "a = bar.func_in_bar()",
    "step_into": "",
    "returned_from": ""
  },
  {
    "code": "cyberbrain.register(x)",
    "next": "",
    "prev": "x = foo.func_in_foo()",
    "step_into": "",
    "returned_from": ""
  }
]