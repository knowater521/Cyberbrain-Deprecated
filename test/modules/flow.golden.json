[
  {
    "code": "x = foo.func_in_foo()",
    "next": "cyberbrain.register(x)",
    "prev": "",
    "step_into": "a = bar.func_in_bar()",
    "returned_from": "return a",
    "param_to_arg": {}
  },
  {
    "code": "a = bar.func_in_bar()",
    "next": "return a",
    "prev": "x = foo.func_in_foo()",
    "step_into": "return 1",
    "returned_from": "return 1",
    "param_to_arg": {}
  },
  {
    "code": "return 1",
    "next": "",
    "prev": "a = bar.func_in_bar()",
    "step_into": "",
    "returned_from": ""
  },
  {
    "code": "return a",
    "next": "",
    "prev": "a = bar.func_in_bar()",
    "step_into": "",
    "returned_from": ""
  },
  {
    "code": "cyberbrain.register(x)",
    "next": "",
    "prev": "x = foo.func_in_foo()",
    "step_into": "",
    "returned_from": ""
  }
]